sample/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,8 @@
         android:theme="@style/AppTheme" >
         <activity
             android:name=".MainActivity"
-            android:label="@string/title_activity_main" >
+            android:label="@string/title_activity_main"
+            android:configChanges="orientation|keyboard|keyboardHidden|screenSize|screenLayout|uiMode" >
```

Comment:
I know you added this to test it, but can you revert it to the original state. I'd like the sample app to reflect the recommended configuration.